backend/canvas_app_explorer/models.py

```diff
@@ -39,6 +39,7 @@ class LtiTool(models.Model):
     privacy_agreement = HTMLField()
     support_resources = HTMLField()
     canvas_placement = models.ManyToManyField(CanvasPlacement)
+    metadata = HTMLField(null=True)
```

Comment:
@jonespm, how do we feel about the name `metadata`, as opposed to just `notes` or `internal_notes`?

Reply:
I could see adding `help_text` here to indicate this isn't shown to users, especially if we don't change the name to internal. Something like this:
```suggestion
    metadata = HTMLField(null=True, help_text='Field for internal notes (not shown to users)')
```

Reply:
Yeah, I think internal_notes sounds like a better field name.

Reply:
Reading thru this request it's not entirely clear to me why we need this field and what will be entered.  Nonetheless, I agree that Internal notes is better than 'metadata'